QubitServer/src/main/java/org/labrad/qubits/channeldata/AnalogData.scala

```diff
@@ -0,0 +1,9 @@
+package org.labrad.qubits.channeldata
+
+import org.labrad.qubits.channels.AnalogChannel
+
+trait AnalogData extends Deconvolvable {
+  def setChannel(channel: AnalogChannel): Unit
+  def getDeconvolved(): Array[Int]
```

Comment:
yeah, you're correct on both counts. I have changes along those lines queued up for later, but I've left the names as they are for now because all the code that calls into these classes is still java.